src/test/store/receive-profile.js

```diff
@@ -0,0 +1,139 @@
+import { assert } from 'chai';
```

Comment:
I mostly added comments to the other files, so you can proofread them if you want , but no obligation